### ISSUE TITLE
Add ability to disable spell checking

### ISSUE THREAD

**Issue by frankdekker** [MicrosoftEdge/WebView2Feedback#584]
By default the WebView2 browser has spellchecking enabled. We're working on an application where input can be done in multiple languages and spellchecking isn't necessary, and will be more of a distraction than it is a benefit.

Example from w3 schools html input:
![image](https://user-images.githubusercontent.com/2179983/97463008-98ac6100-193f-11eb-813c-cd63992a5580.png)

**Request**
Add the ability to disable the spellchecking via the CoreWebView2EnvironmentOptions class. Once set, the browser's spellchecker is disabled.

**Comment by pagoe-msft** [comment]
@frankdekker 

Can this be accomplished by setting "spellcheck" = "False"?

https://www.w3schools.com/Tags/att_global_spellcheck.asp

**Comment by frankdekker** [comment]
> 
> 
> @frankdekker
> 
> Can this be accomplished by setting "spellcheck" = "False"?
> 
> https://www.w3schools.com/Tags/att_global_spellcheck.asp

True, but there's some web content out of our control while still requiring consistent functionality.

**Comment by frankdekker** [comment]
I come from JxBrowser which has the functionality to disable the spell checker, and after some searching it seems that CEF also has support for it via `RequestContext::SetPreference( "browser.enable_spellchecking", false);`  :)

**Comment by pagoe-msft** [comment]
@frankdekker 

I've added this feature request to our backlog. Until we add this functionality, will spellcheck = false unblock you for the time being? 

For web content out of your control, does [ExecuteScript](https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2.executescriptasync?view=WebView2-dotnet-1.0.674-prerelease) work?

**Comment by frankdekker** [comment]
@pagoe-msft Thanks! And I'll fiddle some with ExecuteScript. Thanks for the tip.

**Comment by jasonstephen15** [comment]
Hi @frankdekker, wanted to circle back to this. Were you able to solve your issue with the ExecuteScript workaround?

**Comment by frankdekker** [comment]
@jasonstephen15 Hi, the ExecuteScript solution works partially. You can basicly catch most scenario's but, there always is a form element that you miss. Eg client side generated form fields that are shown are very hard to fix.

So in the end, no it's not a viable workaround, and would love to see an easier way to disable spellcheck.

**Comment by ukandrewc** [comment]
@frankdekker You could get those by implementing a MutationObserver.

**Comment by frankdekker** [comment]
> 
> 
> @frankdekker You could get those by implementing a MutationObserver.

Sure, but technically it's riding a bike with a leaky tire and just keep pumping air in it. I rather have the ability to disable it and not worry about an edge case the script solution missed :).

**Comment by mkb13** [comment]
Not what OP is asking for but I would like to be able to specify the language to do the spellcheck in using the lang attribute (e.g. in a textarea).  Currently the lang attribute does nothing -- spellcheck happens in whatever languages I selected in browser settings for spellcheck (not even for the language of the browser or for the language of the page).

**Comment by Novack** [comment]
I see that this has been an issue for some time now. I've found that is not that cannot be disabled, cannot be done nothing at all with the spell checker. To make it worst, if you need to disabled the native context menus, users are left with no control over the spell checker. This is our case, and we are seeing all kinds of trouble.

I think it goes beyond an option to disable it, so I just opened #3758.
